forastero/bench.py

```diff
@@ -492,7 +509,7 @@ def _do_decorate(func):
         return _do_decorate
 
     @classmethod
-    def parameter(cls, name: str) -> Callable:
+    def parameter(cls, name: str, cast: Callable[[str], Any]) -> Callable:
```

Comment:
With this you either need to update the docs to make it compulsory to set the cast type, or provide a string cast as the default